public/locales/fr/translation.json

```diff
@@ -141,6 +141,12 @@
           "title": "Aucun scénario n'a encore été créé",
           "label": "Vous pouvez créer un scénario en cliquant sur \"Créer un scénario\""
         },
+        "nodashboard": {
+          "label": "Aucun dashboard n'a pas été configuré pour ce type de scénario"
```

Comment:
Typo: a better phrasing would be "Aucun dashboard n'a été configuré pour ce type de scénario" or "Aucun dashboard configuré pour ce type de scénario"